### ISSUE TITLE
A proper tutorial is needed.

### ISSUE THREAD

**Issue by bartekus** [azukiapp/docker-deploy#20]
I think a proper tutorial, that's provider neutral is definitely highly sought after.

There are many facets that are not imminently clear, such as necessity to have ansible installed on top of installing the azk.io
Furthermore, a best practice should include creation of separate azk-service account that would run all aspects of the azk deployment functionality, this perhaps could alleviate some of the current single user deploying the app aspect but again, I'm certainly not an expert so this is just my 2 cents of an opinion.

**Comment by saitodisse** [comment]
Indeed.

We also need to improve README for developers too.

It is difficult to test changes. What should I do after making any changes to test.
I need to build the image?

For example:
How can I test changes made on this PR https://github.com/azukiapp/docker-deploy/pull/19?

**Comment by bartekus** [comment]
I was able to find few article on medium that you guys wrote, I think they are gold and need to be reference in docs if not all together incorporated into them. Any information, is a good information especially from the developers of the azk itself! :D

**Comment by saitodisse** [comment]
> I think they are gold and need to be reference in docs

:+1: Good point -  We were talking about this other day. This is in our backlog.

Thanks for remembering us.

**Comment by bartekus** [comment]
By the way guys, I never thank you for putting up with my never ending questions :D but seriously, huge thank you! 

As we're getting more interest generated in our Meatier project you'll be getting some very deserved exposure from Meteor community as well so I'm happy to make azk know to anybody and everybody that I know. You guys simply ROCK!

**Comment by saitodisse** [comment]
Thanks to you! :heart: :heart: :heart: - I loved the idea behind https://github.com/mattkrick/meatier project.
I stopped using the Meteor because of its monolithic structure. Surely I want to test with azk and maybe put in our http://run.azk.io/

**Comment by bartekus** [comment]
By the way, we are organizing a group of developers/hackers under organization called Araphel which serves as a communication channel and support group - it requires nothing of its members other than for them to pursue their passions / dreams while being the best that they can be. It would be our honor to extend the invites to all of the members of the azk.io team. We currently consist of creators of Meatier, SocketCluster, Structor and DGraph and would love to have you in our circle! Please let me know if you are interested and I'll send invites immediately!

**Comment by saitodisse** [comment]
@Bartekus I spoke with the team here. We'll be honored to be part of Araphel organization.

**Comment by m9dfukc** [comment]
Yes, a proper tutorial would be highly appreciated. Can we please open this ticket again?!

**Comment by saitodisse** [comment]
@m9dfukc We already have some good articles about deploy:
- [How To Deploy a Rails App with azk](https://www.digitalocean.com/community/tutorials/how-to-deploy-a-rails-app-with-azk) (2016-02, at Digital Ocean)
- [[HowTo] “Run Project”: What about deployment?](https://blog.azukiapp.com/howto-run-project-what-about-deployment-8715c92b8403#.r34aevcmh) (2015-09, Medium)
- [Using azk to deploy an application to DigitalOcean](https://medium.com/azuki-news/using-azk-to-deploy-an-application-to-digitalocean-4fe9a5db0356#.nmtjpqoly) (2015-08, at Medium)

@Bartekus - I will keep this issue open to a least remember to reefer this articles on README
and we are going to improve http://docs.azk.io/en/deploy/

**Comment by m9dfukc** [comment]
@saitodisse thanks for the links ... the thing is they are all digital ocean specific. Is there a single tutorial that show the whole process, step by step, for a amazon aws docker deploy?!

**Comment by saitodisse** [comment]
@m9dfukc you welcome. There is no tutorial for any other deploy server besides digital ocean yet. Sorry.
